javascript/code-challenges/leftJoin/README.md

```diff
@@ -0,0 +1,33 @@
+# Join Left
+
+## Code Challenge 33
+
+Authors: Tahmina Ringer
+
+- Write a function that LEFT JOINs two hashmaps into a single data structure.
+- The first parameter is a hashmap that has word strings as keys, and a synonym of the key as values.
+- The second parameter is a hashmap that has word strings as keys, and antonyms of the key as values.
+- Combine the key and corresponding values (if they exist) into a new data structure according to LEFT JOIN logic.
+- LEFT JOIN means all the values in the first hashmap are returned, and if values exist in the “right” hashmap, they are appended to the result row. If no values exist in the right hashmap, then some flavor of NULL should be appended to the result row.
+- The returned data structure that holds the results is up to you. It doesn’t need to exactly match the output below, so long as it achieves the LEFT JOIN logic.
+- Avoid utilizing any of the library methods available to your language.
+
+- INPUT: Two hash tables that contain Keys and values
+- OUTPUT:  An array of all the keys and values of the left table and matching key values from the right table. If no key from right table exists null will be returned.
+
+## EDGE CASES:
+
+Empty hash table
+
+### Big O
+
+- Space -O(1)
+- Time -O(n)
+
+## Approach & Efficiency
+
+I wrote my code first and then wrote tests to validate the code.
+
+### Whiteboard
+
+- [Whiteboard](whiteboard.jpg)
```

Comment:
Looks like you are off to a great start Tahmina! Once you have code and tests written I will regrade for you!